src/boundingboxcomputation.ts

```diff
@@ -10,7 +10,12 @@
 // nvg.js#L449
 
 export class BoundingBoxComputation {
-  constructor(x1, y1, x2, y2) {
+  x1: number;
+  y1: number;
+  protected x2: number;
+  protected y2: number;
```

Comment:
Why mixed visibility here? Can't all be private?

Reply:
Glyph is accessing these attributes:

https://github.com/0xfe/vexflow/blob/b373637eb2ae700a6c6cbe2606448fdea461b317/src/glyph.ts#L281

Reply:
I made to access functions in order to make them protected.